ReferenceApp/src/main/java/com/google/ads/interactivemedia/v3/samples/videoplayerapp/VideoPlayerController.java

```diff
@@ -102,7 +122,26 @@ public void onAdEvent(AdEvent adEvent) {
                   // played. AdsManager.start() begins ad playback. This method is
                   // ignored for VMAP or ad rules playlists, as the SDK will
                   // automatically start executing the playlist.
+
                   adsManager.start();
+
+                  break;
+                case STARTED:
+                  Ad ad = adEvent.getAd();
+                  if (ad.getAdId().contains("truex")) {
```

Comment:
I'm not sure if this is true for all TrueX ads.  I had ended up doing this as a simple debuggy way to set this up.  I could parse the url which is more typical.

Reply:
the convention is to look at getAdSystem instead
e.g. see the HTML5 vast file, and the getAdSystem call